src/pages/RepoPage/FailedTestsTab/FailedTestsTab.spec.tsx

```diff
@@ -1,66 +1,127 @@
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
 import { render, screen } from '@testing-library/react'
-import userEvent from '@testing-library/user-event'
+import { userEvent } from '@testing-library/user-event'
+import { graphql } from 'msw'
+import { setupServer } from 'msw/node'
 import { PropsWithChildren, Suspense } from 'react'
 import { MemoryRouter, Route, useLocation } from 'react-router-dom'
 
 import { useRedirect } from 'shared/useRedirect'
 
 import FailedTestsTab from './FailedTestsTab'
 
-jest.mock('shared/useRedirect')
 jest.mock('./GitHubActions', () => () => 'GitHub Actions tab')
+jest.mock('./CodecovCLI', () => () => 'Codecov CLI tab')
+jest.mock(
+  './FailedTestsTable/FailedTestsTable.tsx',
```

Comment:
Are we safe to remove `FailedTestsTable.tsx` now?

Reply:
Unfortunately not, I think with how the imports are structured for the component we need to keep this syntax. I do agree its a bit ugly tho 🙈